zulipterminal/config/keys.py

```diff
@@ -244,16 +249,16 @@ class KeyBinding(TypedDict):
         'excluded_from_random_tips': True,
         'key_category': 'searching',
     },
+    'EXECUTE_SEARCH': {
+        'keys': ['enter'],
+        'help_text': 'Execute the search and go to search results',
```

Comment:
Nit: We can address this on merging, but 'execute' is likely not the best user-facing language.

Perhaps "Submit search and browse results"?

I don't have the web app to compare to right now (as we discussed in the call).

Reply:
I checked the [list here](https://zulip.com/help/keyboard-shortcuts). They don't seem to use an equivalent for the search op. But, I did find an equivalent for ACTIVATE_BUTTON - "Use Enter to engage with elements".

I have updated the EXECUTE_SEARCH description to match your suggestion, and left the ACTIVATE_BUTTON as-is.